test/plugins/auth/ldap.js

```diff
@@ -0,0 +1,156 @@
+"use strict";
+
+const ldapAuth = require("../../../src/plugins/auth/ldap");
+const Helper = require("../../../src/helper");
+const ldap = require("ldapjs");
+const expect = require("chai").expect;
+
+const user = "johndoe";
+const correctPassword = "loremipsum";
+const wrongPassword = "dolorsitamet";
```

Comment:
Doesn't need to be a const here.

Reply:
Mmh but why not making it const since it can be const?